compiler/rustc_feature/src/unstable.rs

```diff
@@ -199,6 +199,8 @@ declare_features! (
     (internal, prelude_import, "1.2.0", None, None),
     /// Used to identify crates that contain the profiler runtime.
     (internal, profiler_runtime, "1.18.0", None, None),
+    /// Allows the use of scalable SIMD types.
+    (unstable, repr_scalable, "CURRENT_RUSTC_VERSION", None, None),
```

Comment:
This should get a tracking issue and moved to the `feature-group-start: actual feature gates` section.

Reply:
Also if the version you end up landing is not entirely complete, it should be marked as `incomplete` instead of `unstable`.